packages/eliom/eliom.11.0.0/opam

```diff
@@ -0,0 +1,63 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Advanced client/server Web and mobile framework"
+description: """
+Eliom is a framework for implementing Web sites and client/server Web and mobile applications.
+It uses advanced concepts to simplify the implementation of common behaviors (e.g. scoped sessions, continuation based Web programming ...).
+It uses advanced static typing features of OCaml to check many properties of the Web application at compile-time (html, page parameters ...).
+Eliom allows implementing the whole application as a single program that includes both the client and the server code. For example, you can implement event handlers (onclick ...) directly in OCaml, and you can call a server-side OCaml function from the client. Pages are generated either on the server or the client.
+These client-side features remain compatible with traditional Web programming (links, forms, URLs, bookmarks, sessions ...).
+It is possible to generate mobile applications for Android and iOS with the exact same code as your Web application.
+The client-side code is compiled to JS using Ocsigen Js_of_ocaml or to Wasm using Wasm_of_ocaml.
+"""
+maintainer: ["dev@ocsigen.org"]
+authors: ["dev@ocsigen.org"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://ocsigen.org/eliom/"
+bug-reports: "https://github.com/ocsigen/eliom/issues"
+depends: [
+  "dune" {>= "3.6"}
+  "ocaml" {>= "4.08.0"}
```

Comment:
CI is current failing on ocaml versions =< 4.11 due to the injectivity annotation, which apparently was added in 4.12 (e.g., in https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/8a7c2e44a912ad011644f5ad1a9808ceb3a3425b/variant/compilers,4.08,eliom.11.0.0):

```
#=== ERROR while compiling eliom.11.0.0 =======================================#
# context              2.3.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.11.2 | pinned(https://github.com/ocsigen/eliom/archive/refs/tags/11.0.0.tar.gz)
# path                 ~/.opam/4.11/.opam-switch/build/eliom.11.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p eliom -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/eliom-7-505e61.env
# output-file          ~/.opam/log/eliom-7-505e61.out
### output ###
# (cd _build/.sandbox/5e828d3fdeceadb53ee04631f1c5620d/default && .ppx/13d6111584a1a9a5b7ca130081cf0d43/ppx.exe --cookie 'library-name="eliom_server"' -o src/lib/server/eliommod.pp.mli --intf src/lib/server/eliommod.mli -corrected-suffix .ppx-corrected -diff-cmd - -dump-ast)
# File "src/lib/server/eliommod.mli", line 25, characters 7-8:
# 25 |   type !'a t
#             ^
# Error: Syntax error: 'end' expected
# File "src/lib/server/eliommod.mli", line 23, characters 11-14:
# 23 | module S : sig
#                 ^^^
#   This 'sig' might be unmatched



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+- The following actions failed
| - build eliom 11.0.0
```

So I think you'll either need to drop that annotation or lift the lower bound here:

```suggestion
  "ocaml" {>= "4.12"}
```